apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.ts

```diff
@@ -43,23 +45,15 @@ function prettyCourseName(ltiClaim: Lti13Claim) {
 
 async function courseInstancesAllowedToLink({
   course,
-  user_id,
-  authn_user_id,
-  is_administrator,
-  authn_is_administrator,
+  authzData,
 }: {
   course: Course;
-  user_id: string;
-  authn_user_id: string;
-  is_administrator: boolean;
-  authn_is_administrator: boolean;
+  authzData: PageAuthzData;
 }) {
   const course_instances = await selectCourseInstancesWithStaffAccess({
     course,
-    user_id,
-    authn_user_id,
-    is_administrator,
-    authn_is_administrator,
+    authzData,
+    requiredRole: ['Previewer', 'Student Data Viewer'],
```

Comment:
This is my fallback for a staff page. This PR intends to add the machinery for performing this sort of authentication, but not tightening authentication yet.